spec/tictactoe_spec.rb

```diff
@@ -7,7 +7,22 @@
   let (:test_ttt) { TicTacToe.new(game_io, player1, player2) }
 
   it "runs the game loop " do
-    allow(test_ttt.game_loop).to receive(:game_loop) { }
-    test_ttt.game_loop
+    expect(test_ttt.game_loop).to receive(:game_loop).once
+    expect(test_ttt).to receive(:play_again?).and_return(false)
+    test_ttt.play_game
+  end
+
+  describe "the play_again? method " do
```

Comment:
I'd prefer `play_again?` be private since no other class depends on it. This would mean that the behavior needs to be tested higher up inside the game loop. Were you able to figure out the RSpec multiple calls feature? It'll be necessary here to prevent an infinite loop...